database/migrations/2024_04_01_133925_orders.php

```diff
@@ -10,7 +10,7 @@
     public function up(): void
     {
         Schema::create('orders', function (Blueprint $table) {
-            $table->id();
+            $table->id()->primaryKey;
```

Comment:
Remove unnecessary method call to `->primaryKey`, because `->id()` already sets the column to a primary key in the db.